e2e/steps_file.js

```diff
@@ -67,6 +67,7 @@ module.exports = function () {
 
     signOut() {
       this.click('Sign Out');
+      this.wait(2); // in seconds
```

Comment:
Waiting for? Again, couldn't we do this a bit nicer? Maybe we could open Tech debt to get your changes over the line quicker and ensure we are getting rid of those wait(2)?